magento/magento2
# Fix incorrect row totals in invoice email

### Description (*)

Invoice confirmation email showing incorrect row totals when partially cancelled as it displays amounts from order item, not from invoice item.

### Fixed Issues

1. Fixes magento/magento2#32307

### Manual testing scenarios (*)

1. Add few products to the cart with multiple qty
2. Proceed to checkout and complete the order
3. Invoice the order for partial qty in backend
4. Ship the same qtys as per invoice
5. Cancel the rest of order
6. (optional) Go to invoice view in the backend and click "Send Email" if automatic emails are disabled
7. Check whether invoice email received contains correct row totals

### Contribution checklist (*)
